### DISCUSSION TITLE
Will 1.0 be supported?

### DISCUSSION THREAD

**Discussion by Yanni-X** [wolveix/satisfactory-server#251] [Q&A] [answered]
Hey hey, quickly question: will 1.0 be supported?

As far as I understood the image, it pulls the latest version of the game on each startup. Therefore it should be straightforward to start the server with 1.0 (unless there are some breaking changes in 1.0 not known of yet I guess?)

would be great to have an estimate :)
Best regards

**Comment by wolveix** [comment, marked as answer]
Hey @Yanni-X!

Yes, absolutely, 1.0 should work as-is on release. If not, I'll be here to straighten things out :)

**Reply by tsampugna** [reply]
Really cool to hear!
I saw a video from the community manager stating that the ports may change, "everything through TCP and UDP 7777". However I can't help but notice the wiki remains the same. 

Source: https://youtu.be/v8piXNQwcUw?si=6pjeqZpFDIHkKn_3&t=224
But any port changes would be easy to make. Launch is at 5 pm CEST! See you all there!

**Reply by Yanni-X** [reply]
And some seem to have problems with the dedicated server in generall, does this maybe also apply to the container? Something about the  -multihome parameter that has to be removed

https://www.reddit.com/r/SatisfactoryGame/comments/1fdkk83/whats_the_current_way_of_hosting_dedicated/

**Reply by wolveix** [reply]
Doesn't make a difference unfortunately
